Modules/Scripted/SampleData/SampleData.py

```diff
@@ -269,6 +269,27 @@ def setup(self):
         if self.developerMode is False:
             self.setCategoryVisible(self.logic.developmentCategoryName, False)
 
+        customFrame = ctk.ctkCollapsibleGroupBox()
+        self.categoryLayout.addWidget(customFrame)
+        customFrame.title = _("Custom data download")
```

Comment:
For the title, I suggest to use something more specific like "Load data from URL"

```suggestion
        customFrame.title = _("Load data from URL")
```

Reply:
Or even `Add data from URL`. 


This would be consistent with the existing "File -> Add Data" functionality